### ISSUE TITLE
tdlib version 1.7.9 is required

### ISSUE THREAD

**Issue by zz64** [ars3niy/tdlib-purple#144]
Hi, any solution to build this plugin with tdlib 1.8.0 ?

```
* Fetching https://github.com/ars3niy/tdlib-purple.git ...
git fetch https://github.com/ars3niy/tdlib-purple.git +HEAD:refs/git-r3/HEAD
remote: Enumerating objects: 3301, done.
remote: Counting objects: 100% (224/224), done.
remote: Compressing objects: 100% (146/146), done.
remote: Total 3301 (delta 117), reused 166 (delta 78), pack-reused 3077
Réception d'objets: 100% (3301/3301), 1.65 Mio | 1.00 Mio/s, fait.
Résolution des deltas: 100% (2344/2344), fait.
Depuis https://github.com/ars3niy/tdlib-purple
....
-- The CXX compiler identification is GNU 11.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/x86_64-pc-linux-gnu-g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:11 (message):
  tdlib version 1.7.9 is required

  (version found: 1.8.0)
```
Thanks

**Comment by BenWiederhake** [comment]
Yes, you're very welcome to port tdlib-purple to tdlib 1.8.0, and I'd be happy to review and possibly merge a PR. Until then, using version 1.7.9 suffices, in particular commit [9f6b3699](https://github.com/tdlib/td/commit/9f6b3699c6496e911850046d8b6b6db973911737) (see [PR](https://github.com/ars3niy/tdlib-purple/issues/133))

**Comment by heeplr** [comment]
I'm also failing to compile against tdlib 1.8.0 and 1.7.0.

Where does 1.7.9 come from? I can't find the release [tag](https://github.com/tdlib/td/tags) or anything in the [changelog](https://github.com/tdlib/td/blob/master/CHANGELOG.md). 

tdlib-purple [seems to require](https://github.com/ars3niy/tdlib-purple/blob/master/CMakeLists.txt#L9) *exactly* that version. Is that really needed or will lower microversions also work?

**Comment by BenWiederhake** [comment]
You might want to use this PR: https://github.com/ars3niy/tdlib-purple/pull/159

**Comment by heeplr** [comment]
@BenWiederhake Thanks. I worked around the problem by using commit 42aadccc8b (last 1.7.9 commit) as anchor.
I'm using gentoo and live builds can be "abused" that way but I suppose best practice for packaging would be to only use official releases (or ignoring the microversion during checks until problems arise and a specific microversion needs to be fixed until the problems are solved).

I will compile manually and give PR #159 a try with tdlib 1.8.0.
